src/webui/www/private/scripts/mocha-init.js

```diff
@@ -915,6 +916,31 @@ const initializeWindows = function() {
         return names.join("\n");
     };
 
+    copyInfohashFN = function(policy) {
+        const selectedRows = torrentsTable.selectedRowsIds();
+        const infohashes = [];
+        if (selectedRows.length) {
+            const rows = torrentsTable.getFilteredAndSortedRows();
+            if (policy === 1) {
```

Comment:
Maybe it is better to use `switch` to match the code to c++.

Reply:
>Maybe it is better to use `switch` to match the code to c++.

Maybe you change JS code later since you seem more familiar with it? I would prefer never deal with it.

Reply:
> 
> 
> Maybe it is better to use `switch` to match the code to c++.

OK, changed. `switch` seems to exist in JS after all.